new/CIA20024.xml

```diff
@@ -0,0 +1,927 @@
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i2.2.31">
+                                <locus from="135r" to="137r"/>
+                                <textLang mainLang="gez"/>
+                                <title type="complete" ref="LIT5673AkkonuColl#AkkColl31"/>
+                                
+                                <msItem xml:id="ms_i2.2.31.1">
+                                    <locus from="135ra" to="136vb"/>
+                                    <title type="complete" ref="LIT3617Miracle"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i2.2.31.2">
+                                    <locus from="136vb" to="136vc"/>
+                                    <title type="complete" ref="LIT4274AkkonuB#Stanza31"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i2.2.31.3">
+                                    <locus target="#137r"/>
+                                    <title type="complete" ref="LIT5673AkkonuColl#AkkMin31"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i2.2.32">
+                                <locus from="137va" to="139r"/>
+                                <textLang mainLang="gez"/>
+                                <title type="complete" ref="LIT5673AkkonuColl#AkkColl32"/>
+                                
+                                <msItem xml:id="ms_i2.2.32.1">
+                                    <locus from="137va" to="138rb"/>
+                                    <title type="complete" ref="LIT3618Miracle"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i2.2.32.2">
+                                    <locus from="138rb" to="138rc"/>
+                                    <title type="complete" ref="LIT4274AkkonuB#Stanza32"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                
+                                
+                                <msItem xml:id="ms_i2.2.32.3">
+                                    <locus target="#139r"/>
+                                    <title type="complete" ref="LIT5673AkkonuColl#AkkMin32"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                <msItem xml:id="ms_i2.2.33">
```

Comment:
This item is included in ms_i2.2.32, but numbered ms_i2.2.33: It should either be numbered ms_i2.2.32.4 or be moved outside of ms_i2.2.32. Both solutions would be fine, you have to decide which is conceptually better.